app/javascript/custom/pagination.js

```diff
@@ -0,0 +1,13 @@
+jQuery(function() {
```

Comment:
when to use jQuery() function ?? Do $(document.ready (function()) doesn't solve the problems ?? What will be the problem if we use document.ready or simply $() ?? o

Reply:
$ is an alias for jQuery but many libraries use $ to access functions. So if at all we are using more than one javascript library then it is better to use jQuery to avoid conflicts.